schedule/__init__.py

```diff
@@ -226,6 +226,9 @@ def __init__(self, interval: int, scheduler: Scheduler = None):
         # optional time at which this job runs
         self.at_time: Optional[datetime.time] = None
 
+        # optional time zone of the self.at_time field. Only relevant when at_time is not None
+        self.at_time_zone = None
```

Comment:
I wanted to ask about the optional but the timezone is already set as an option. It can be used without timezone example:

```python
 every().day.at("06:30")
```

If need to timezone just use like:

```python
every().day.at("10:30", "America/New_York")
```